packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -229,10 +229,11 @@ export class SingleAddressWallet implements ObservableWallet {
     this.transactions = createTransactionsTracker({
       addresses$,
       chainHistoryProvider: this.chainHistoryProvider,
+      inFlightTransactionsStore: stores.inFlightTransactions,
       newTransactions: this.#newTransactions,
       retryBackoffConfig,
-      store: stores.transactions,
-      tip$: this.tip$
+      tip$: this.tip$,
+      transactionsStore: stores.transactions
```

Comment:
Even better:
```suggestion
      transactionsHistoryStore: stores.transactions
```

Reply:
I would like to do a rebase on top of master and a squash, before doing any other changes. Is that ok?

Reply:
Thank you @mkazlauskas .
Stores tests are here: [bb28d4f](https://github.com/input-output-hk/cardano-js-sdk/pull/293/commits/bb28d4f32e00b88b87a269be5472c44bdcaa8172)
Rename is here: [86cfb5e](https://github.com/input-output-hk/cardano-js-sdk/pull/293/commits/86cfb5ed9ab68e3a3751c90e397c9aefe9b8a064)